templates/deb/postinst_upgrade.sh.erb

```diff
@@ -8,18 +8,26 @@ after_upgrade() {
 <%=  script(:after_upgrade) %>
 <% end -%>
 
-<% if attributes[:deb_systemd] -%>
+<%# if any systemd services specified, loop through and start them -%>
+<% unless attributes[:deb_systemd].empty? -%>
 systemctl --system daemon-reload >/dev/null || true
-if ! systemctl is-enabled <%= attributes[:deb_systemd] %> >/dev/null 
+sysctl=$(command -v deb-systemd-invoke || echo systemctl)
+<% attributes[:deb_systemd].each do |service| -%>
+if ! systemctl is-enabled <%= service %> >/dev/null 
```

Comment:
Did you mean `"$sysctl"` here instead of `systemctl` ? Also, for variable names, `sysctl` may confuse future maintainers as `sysctl` is a different tool from `systemctl`. This comment applies to any invocations of `systemctl` where `$sysctl` seems the appropriate replacement.

Reply:
good find, I've changed it `debsystemd` instead